packages/mask/.webpack/manifest.ts

```diff
@@ -32,6 +32,8 @@ function modify(manifest: Manifest, flags: NormalizedFlags) {
         manifest.name += ' (Beta)'
     } else if (flags.channel === 'insider') {
         manifest.name += ' (Nightly)'
+    } else if (flags.channel === 'stable') {
+        stableDevelopmentExtensionID(manifest)
```

Comment:
Why you're going to do this? This will make our extension rejected by the Chrome Store.